src/test/java/TrainingTimeManagerTest.java

```diff
@@ -0,0 +1,103 @@
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDateTime;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class TrainingTimeManagerTest {
+
+    /* calculateHoursBetweenTwoDateTimes method: */
+    @Test
+    public void calculateHoursBetweenTwoDateTimes_SameDayTimeDifference() {
+        assertEquals(7, TrainingTimeManager.calculateHoursBetweenTwoDateTimes(
+                LocalDateTime.of(2021, 9, 6, 10, 0),
+                LocalDateTime.of(2021, 9, 6, 17, 0)
+        ));
+    }
+
+    @Test
+    public void calculateHoursBetweenTwoDateTimes_OutsideWorkingHours() {
+        assertEquals(8, TrainingTimeManager.calculateHoursBetweenTwoDateTimes(
+                LocalDateTime.of(2021, 9, 6, 7, 0),
+                LocalDateTime.of(2021, 9, 6, 19, 0)
+        ));
+    }
```

Comment:
these tests look similar, the only difference is in the test data, so theoretically they might be simplified, you might want to check parametrized tests in Junit